documentation/asciidoc/computers/raspberry-pi/boot-http.adoc

```diff
@@ -0,0 +1,113 @@
+== HTTP Boot
+
+The network install feature uses HTTP over ethernet to boot the device into the xref:getting-started.adoc#using-raspberry-pi-imager[Raspberry Pi Imager] application. You can change how this is configured and make use of HTTP Boot for your own purposes.
+
+=== Requirements
+
+To use HTTP boot you need to use the LATEST / STABLE bootloader configuration, and update to the bootloader dated on or after 10 March 2022. HTTP boot only works over ethernet, so you need an Ethernet cable connected to the network, e.g. via a socket on the back of your router.
+
+=== Configuration Properties
+
+==== NET_INSTALL_ENABLED
+
+By default net install is enabled on Pi4 and Pi400. On CM boards it is disabled by default. 
+
+When net install is enabled it detects a keyboard and displays the Net Install screen on boot. You can switch this behaviour on and off with this property.
+
+To enable network install add, `NET_INSTALL_ENABLED=1`, or to disable network install add `NET_INSTALL_ENABLED=0`.
+
+This setting is ignored and net install / keyboard detection on boot is disabled if `DISABLE_HDMI=1` is set.
+
+==== NET_INSTALL_KEYBOARD_WAIT
+
+If net install is enabled, the bootloader waits to detect a keyboard and the `SHIFT` key to initiate network install. You can change the length of this wait in milliseconds with this property,
+
+Default: `900`
+
+Adding `NET_INSTALL_KEYBOARD_WAIT=0` disables the keyboard wait, although network install can still be initiated if no boot files are found and USB boot `4` is in `BOOT_ORDER`.
+
+NOTE: Testing suggests Keyboard / shift detection takes at least 750ms.
+
+==== HTTP_HOST
+
+If a HTTP download is initiated boot.img and boot.sig are downloaded from this server.
+
+Default: `fw-download-alias1.raspberrypi.com`
+
+Invalid host names will be ignored.
+If `HTTP_HOST` is set then HTTPS is disabled and plain HTTP used instead.
+You should be able to specify an IP address and avoid the need for a DNS lookup.
```

Comment:
"and avoid" -> "to avoid"?
And I'm not sure I like the vagueness of "You should be able to specify..." - could this be changed to "You can specify..." ?